taxonomies/food/categories.txt

```diff
@@ -56393,11 +56399,11 @@ es: Harinas de castañas T110
 fr: Farines de châtaigne T110
 hr: Brašna od kestena t110
 
-< en:Dried fruits
-< en:Nuts and their products
-en: Nuts with dried fruits, Nuts and dried fruits
+< en:Plant-based foods
+< en:Snacks
```

Comment:
It feels like `< en:Dried fruits` + `< en:Nuts and their products` is a more accurate parent than `< en:Plant-based foods` (which should already be an ancestor of both of those other ones). Is there a reason to make it more generic here? Also, are all instances of this category meant as snacks (as opposed to, say, müsli, toppings, etc.)?